internal/rest/avi_obj_vs.go

```diff
@@ -254,8 +254,20 @@ func (rest *RestOperations) AviVsBuild(vs_meta *nodes.AviVsNode, rest_method uti
 			}
 			vs.L4Policies = l4Policies
 		}
+		if vs_meta.DefaultPool != "" {
```

Comment:
is it required to add the DefaultPool to checksum calculation?

Reply:
This is for the avi vs model build. Its added because with introduction of l4 ssl either default pool will be set.